index.js

```diff
@@ -0,0 +1,75 @@
+function playRound(playerSelection, computerSelection) {
+  playerSelection = playerSelection.toLowerCase();
+  computerSelection = computerSelection.toLowerCase();
+  if (playerSelection === "rock") {
+    if (computerSelection === "rock") {
+      return "None";
+    } else if (computerSelection === "papper") {
+      return "Lose";
+    } else {
+      return "Win";
+    }
+  } else if (playerSelection === "papper") {
+    if (computerSelection === "papper") {
+      return "None";
+    } else if (computerSelection === "rock") {
+      return "Win";
+    } else {
+      return "Lose";
+    }
+  } else {
+    if (computerSelection === "papper") {
+      return "Win";
+    } else if (computerSelection === "rock") {
+      return "Lose";
+    } else {
+      return "None";
+    }
+  }
```

Comment:
Use a nested switch here instead of nested if - else. Switch offers better readability.